app/controllers/outcome_groups_api_controller.rb

```diff
@@ -319,7 +319,7 @@ def destroy
 
   # @API List linked outcomes
   #
-  # List the immediate OutcomeLink children of the outcome group. Paginated.
+  # A paginated list of the immediate OutcomeLink children of the outcome group. Paginated.
```

Comment:
You should remove the mention of "Paginated" that already exists at the end of this line.